Description: Ensure directory doesn't exist for test_config_dir_doesnt_exist().
Author: Corey Bryant <corey.bryant@canonical.com>
Forwarded: https://review.openstack.org/365153

--- a/oslo_config/tests/test_cfg.py
+++ b/oslo_config/tests/test_cfg.py
@@ -2909,7 +2909,8 @@
         self.assertEqual('whistle-11', self.conf.snafu.bell)
 
     def test_config_dir_doesnt_exist(self):
-        tmpdir = '/tmp/foo'
+        tmpdir = tempfile.mkdtemp()
+        os.rmdir(tmpdir)
 
         self.assertRaises(cfg.ConfigDirNotFoundError,
                           self.conf,
